packages/libssh.rb

```diff
@@ -4,46 +4,51 @@ class Libssh < Package
   source_sha256 'acffef2da98e761fc1fd9c4fddde0f3af60ab44c4f5af05cd1b2d60a3fa08718'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-1_armv7l/libssh-0.9.5-1-chromeos-armv7l.tar.xz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-1_armv7l/libssh-0.9.5-1-chromeos-armv7l.tar.xz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-1_i686/libssh-0.9.5-1-chromeos-i686.tar.xz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-1_x86_64/libssh-0.9.5-1-chromeos-x86_64.tar.xz'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-2_armv7l/libssh-0.9.5-2-chromeos-armv7l.tpxz',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-2_armv7l/libssh-0.9.5-2-chromeos-armv7l.tpxz',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-2_i686/libssh-0.9.5-2-chromeos-i686.tpxz',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/libssh/0.9.5-2_x86_64/libssh-0.9.5-2-chromeos-x86_64.tpxz'
   })
   binary_sha256({
-    aarch64: '7823ea3948867ed6b44ff1d62334aa011db8d3e07452c5a6312f5873a26d681c',
-     armv7l: '7823ea3948867ed6b44ff1d62334aa011db8d3e07452c5a6312f5873a26d681c',
-       i686: '07e75554e44aaef4210d9bf6a17b098953d03054ff5f2614dc4c4719d6936d8b',
-     x86_64: 'a3f1c7727f88271291316d186698e2c94afd275493f3b27e93f2eb152679970e'
+    aarch64: 'fe56c352d678f004866c27bf5b4dc1d05114387f65bc07968834ddbd5ec9bb79',
+     armv7l: 'fe56c352d678f004866c27bf5b4dc1d05114387f65bc07968834ddbd5ec9bb79',
+       i686: 'd524f5485303306cb39720c09e9379a3e636a1f18ba68a547dd6534c762c5c12',
+     x86_64: 'abb34682d2e053ff8a7ca1f2d64d45402127a3460314a0b330641715dff21704'
   })
 
   depends_on 'libgcrypt'
 
   def self.build
-    Dir.mkdir 'builddir'
-    Dir.chdir 'builddir' do
-      system "env CFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
-        CXXFLAGS='-pipe -fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
-        LDFLAGS='-fno-stack-protector -U_FORTIFY_SOURCE -flto=auto' \
-        cmake -G Ninja \
-        #{CREW_CMAKE_OPTIONS} \
-        -DWITH_STACK_PROTECTOR_STRONG=NO \
-        -DWITH_STACK_CLASH_PROTECTION=NO \
-        -DWITH_STACK_PROTECTOR=NO \
-        -DWITH_GCRYPT=ON \
-        .."
+    FileUtils.mkdir('builddir')
+    Dir.chdir('builddir') do
+      system "cmake #{CREW_CMAKE_OPTIONS} \
+      -DWITH_EXAMPLES=OFF \
+      -DBUILD_SHARED_LIBS=OFF \
+      -DWITH_STATIC_LIB=ON \
+      ../ -G Ninja"
+    end
+    system 'ninja -C builddir'
+    Dir.chdir('builddir') do
+      FileUtils.cp 'src/libssh.a', '../' if File.exist?('src/libssh.a')
+      system "cmake #{CREW_CMAKE_OPTIONS} \
+      -DWITH_EXAMPLES=OFF \
+      -DBUILD_SHARED_LIBS=ON \
+      -DWITH_STATIC_LIB=OFF \
```

Comment:
No static library?

Reply:
See the `FileUtils.cp 'src/libssh.a', '../' if File.exist?('src/libssh.a')` bit there?

Reply:
You wouldn't have to do any of that if you enabled static, no?